openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -31,6 +30,12 @@
 
 log = logging.getLogger(__name__)
 
+# Loading pynliner is actually quite expensive because it uses cssutils, which
+# has an expensive initialization. All told, it accounts for 15-20% of "fast"
+# LMS startup (without asset compilation). So we're going to set this as `None`
+# to start, and then lazily replace with the actual import when it's needed.
+pynliner = None  # pylint: disable=invalid-name
```

Comment:
You know, you're right. I was being over-paranoid here because I'm projecting from a past thing I did where the (tiny) overhead did matter. But this happens way too infrequently for that to be a serious consideration -- I'll do the in-function import.